health-check-tcp/main.tf

```diff
@@ -0,0 +1,18 @@
+# [START cloudloadbalancing_health_check_tcp_with_logging]
+resource "google_compute_health_check" "health-check-tcp-with-logging" {
```

Comment:
Please use underscores instead of dashes in resource names

Reply:
Replaced dashes with underscores